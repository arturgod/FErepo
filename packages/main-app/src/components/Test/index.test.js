jest.mock('@ecovadis/shared', () => ({
  Button: ({ children }) => <div>{children}</div>,
}));

import React from 'react';
import { shallow } from 'enzyme';
import { Test } from './';

describe('Button', () => {
  const requiredProps = {
    onClick: jest.fn(),
  };
  it('should exist', () => {
    const wrapper = shallow(<Test {...requiredProps} />);
    expect(wrapper).toMatchSnapshot();
  });
});
