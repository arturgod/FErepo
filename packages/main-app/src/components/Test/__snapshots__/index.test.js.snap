// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button should exist 1`] = `
<Button
  onClick={[Function]}
>
  testButton
</Button>
`;
